app/models/route.rb

```diff
@@ -11,7 +11,7 @@ class Route < ApplicationRecord
 
   scope :authorized, ->(user) { user.has_role?(:superadmin) ? all : with_role(:developer, user) }
 
-  validates :name, presence: true, length: {minimum: 2, maximum: 255}
+  validates :name, uniqueness: true, presence: true, length: {minimum: 2, maximum: 255}
```

Comment:
@bosulliv42 
Where is the "scope"?
Two routes from two different proxies may have the same name.
Did you forget to commit and push ?